### PR TITLE
Update libraries, x-checker things

### PR DESCRIPTION
**Pull request by lionirdeadman**
I believe none of these changes should break anything.

The x-checker will alert you about 6.0.2 but that requires a newer setuptool than given by the runtime, I can remove that if you want.

### CODE REVIEW

File: com.spotify.Client.json [hunk 1]
```diff
@@ -53,7 +53,12 @@
                 {
                     "type": "archive",
                     "url": "https://download.gnome.org/sources/libnotify/0.7/libnotify-0.7.9.tar.xz",
-                    "sha256": "66c0517ed16df7af258e83208faaf5069727dfd66995c4bbc51c16954d674761"
+                    "sha256": "66c0517ed16df7af258e83208faaf5069727dfd66995c4bbc51c16954d674761",
+                    "x-checker-data": {
+		        "type": "gnome",
+			"name": "libnotify",
```
Comment (barthalion) [inline com.spotify.Client.json:59]: If there's one release of libnotify a year, it's a miracle. I don't think it makes sense.

**Comment by barthalion** [comment]
> The x-checker will alert you about 6.0.2 but that requires a newer setuptool than given by the runtime, I can remove that if you want.

Yes, let's avoid unnecessary notifications please.

File: com.spotify.Client.json (hunk 1 above)
Comment (lionirdeadman) [reply com.spotify.Client.json:59]: Doesn't it make even more sense because people will forget that the releases are there?

**Comment by lionirdeadman** [comment]
> > The x-checker will alert you about 6.0.2 but that requires a newer setuptool than given by the runtime, I can remove that if you want.
> 
> Yes, let's avoid unnecessary notifications please.

Actually, turns out I was wrong. 6.0.2 is fine as can be seen on https://github.com/flathub/com.discordapp.Discord/pull/150 . I'm not sure what made me believe that it wasn't in the first place. I'll update to 6.0.2 and python-xlib.

**Comment by barthalion** [comment]
It definitely does not work for me.

image

**Comment by barthalion** [comment]
Oh, false alarm. Looks like it started to work now.

**Pull request merged**